quickstarts/microsoft.network/vnet-create-with-diagnostic-logs/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vnetName": {
```

Comment:
remove this so we can test the defaultValue on the param

Reply:
done